packages/bitcoin/redux/constants.js

```diff
@@ -7,6 +7,7 @@ export const DUCK_BITCOIN = 'bitcoin'
 
 
 export const BITCOIN_CREATE_WALLET = 'BITCOIN/CREATE_WALLET'
+export const BITCOIN_UPDATE_BALANCE = 'BITCOIN/BALANCE/UPDATE'
```

Comment:
Naming conventions here a bit different:
DOMAIN_DUCK/OTHER_PART_OF_CONSTANT
So in this case it should be BITCOIN/BALANCE_UPDATE